util/util.py

```diff
@@ -2,7 +2,7 @@
 import numpy as np
 import pandas as pd
 
-from run_ml import CONSTANT__OUTSIDE
+CONSTANT__OUTSIDE = "OUTSIDE"
```

Comment:
why is this newly defined instead of importing it (or importing this in the other file)

Reply:
I defined it in this file because the run_ml.py is not included in the Docker container of the gates-runtime.
Now, I changed the run_ml.py, so that here `CONSTANT__OUTSIDE` is imported from util.

Reply:
works for me